TOC/Proposal/README.md

```diff
@@ -1 +1 @@
-This folder holds proposals that require TOC members to make decisions. The decision-making process follows the voting rules in TiDB community.
+This folder holds proposals that require TOC members to make decisions. The decision-making process follows the [voting rules](./voting-rules.md) in TiDB community.
```

Comment:
I mean you can just inline the content of voting rules here, because we don't have to set a placement holder README when it can deliver something essential.

Reply:
OK, thanks.